src/CMakeLists.txt

```diff
@@ -15,15 +26,20 @@ target_compile_definitions(journald
         -DG_LOG_DOMAIN="eventd-journald")
 target_link_libraries(journald
     PRIVATE
-        ${EventdEvent_LIBRARIES}
+        ${Eventd_LIBRARIES}
         ${EventdPlugin_LIBRARIES}
+        ${EVENTDHELPERS_LIBRARY}
         ${SystemdJournal_LIBRARIES})
 set_target_properties(journald
     PROPERTIES
         PREFIX    ""
         NO_SONAME 1)
 
+pkg_config_variable(libeventd-plugin pluginsdir EVENTD_PLUGINS_DIR_INTERNAL)
+set(EVENTD_PLUGINS_DIR ${EVENTD_PLUGINS_DIR_INTERNAL}
```

Comment:
Hmm. That breaks installing this to a different prefix than eventd itself, but I don't know that I've ever personally done such a thing for applications with plugins (versus modules like Python).